tests/topotests/lib/pim.py

```diff
@@ -55,7 +55,7 @@ def create_pim_config(tgen, topo, input_dict=None, build=False, load_config=True
     input_dict = {
         "r1": {
             "pim": {
-                "disable" : ["l1-i1-eth1"],
```

Comment:
I'm no expert here, but this looks like accidental removal (also code below). How do you disable PIM from an interface? Is there a reset somewhere else? I only see the two tests below modified (assuming there is more) `test_multicast_pim_sm_topo1.py` and `test_multicast_pim_sm_topo3.py`.

This seems to be doing more than the commit suggests.

Reply:
This was only used in a couple places and is a weird way to do "no pim" on the interface. You have to specify the exact same information as it requires to just turn pim off on the interface directly which other places in the code were already doing so I made things consisten.. One example in this set of changes:

https://github.com/FRRouting/frr/pull/9218/files/53cb4e70ad95359defd341494a5f86929b0f6264..691ea7cda0469c52aeaaf8e32380775377b134eb#diff-a60e553312a4f21704cdf9e8dd4e47800d5c139d2e0f5c3c55567ebe4dac25b7L1554-L1559